pkg/controller/controller.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/kubevirt/containerized-data-importer/pkg/common"
+	. "github.com/kubevirt/containerized-data-importer/pkg/utils/errors"
```

Comment:
Q: do we also want stack traces in controller _main_ (4 times), even if the errors are fatal?